src/renderer/assets/styles/base/_variables.scss

```diff
@@ -51,7 +51,8 @@ $spacing-rel-large: 6vw;
 $border-radius-button: 10px;
 $border-radius-input: 5px;
 $box-shadow: 0.5px 0.5px 1px 1px rgba(0, 0, 0, 0.18);
-$box-shadow-inset: inset 0.5px 0.5px 1px 1px rgba(0, 0, 0, 0.13);
+$box-shadow-inset: 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06);
```

Comment:
In my opinion, input fields should have an inset shadow, this makes it clear to the user that they're editable.

Reply:
Yup. I've only added this so the editor could pop up a bit more. I forgot about the inputs.